Pilot/MessageSender.py

```diff
@@ -15,6 +15,11 @@
 
 import Queue
 import logging
+import wasser as requests
```

Comment:
add check of python version.
If python 2.7 import requests
if python 2.6 import wasser as requests